staticaddr/withdraw/manager.go

```diff
@@ -710,17 +745,13 @@ func (m *Manager) toPrevOuts(deposits []*deposit.Deposit,
 }
 
 func (m *Manager) republishWithdrawals() error {
-	for withdrawalAddress, deposits := range m.activeWithdrawals {
-		address, err := btcutil.DecodeAddress(
-			withdrawalAddress, m.cfg.ChainParams,
-		)
-		if err != nil {
-			log.Errorf("Error decoding withdrawal address: %v", err)
-
-			return err
+	for _, finalizedTx := range m.finalizedWithdrawalTxns {
+		if finalizedTx == nil {
+			log.Warnf("Finalized withdrawal tx is nil")
+			continue
```

Comment:
`m.finalizedWithdrawalTxns` is not persisted, so tx is not re-published if the daemon restarts. IIUC, before this commit it would recreate a transaction and re-publish it.

Reply:
`m.finalizedWithdrawalTxns` is populated by `createFinalizedWithdrawalTx` which is also called in `recover`, so it will still be republished here.